source/declarations.tex

```diff
@@ -993,7 +993,7 @@
 \indextext{specifier!\idxcode{inline}}
 
 \pnum
-The \tcode{inline} specifier can be applied only to the declaration
+The \tcode{inline} specifier shall be applied only to the declaration
 or definition of a variable or function.
```

Comment:
For consistency with the above cases, and for simplicity, remove the "or definition" here?

Reply:
Fixed, also in [dcl.align].